interfaces/http/server/interfaces.go

```diff
@@ -23,6 +23,7 @@ const (
 
 // ListenInfo defines port info
 type ListenInfo struct {
+	Network string        `json:"network"`
```

Comment:
What is the use case here?

Reply:
The 'Network' field is used by getGRPCAddress to decide which address format should be return.